website/content/en/status/report-2023-01-2023-03/drm-drivers.adoc

```diff
@@ -16,8 +16,8 @@ Therefore this status report entry will cover more than just the last quarter.
 
 ==== Update to Linux 5.15 LTS and Linux 5.16
 
-As of this status report, the package:graphics/drm-kmod[] meta port still installs the DRM drivers from Linux 5.10 (released on December 13, 2020).
-This version lacks support for recent GPUs, in particular Intel 12th gen Alder Lake ones.
+As of this status report, the package:graphics/drm-kmod[] meta port still installs the DRM drivers from Linux 5.10 (released on December 13, 2020) on FreeBSD 13.1 and greater.
```

Comment:
@evadot @dumbbell would you like to mention 13.1 (or `1301000`) here, as suggested? 

For less knowledgeable readers to _not_ wrongly assume that `drm-510-kmod` is (also) a default on 12.4.